src/TextField/EnhancedTextarea.js

```diff
@@ -126,7 +126,9 @@ class EnhancedTextarea extends Component {
   }
 
   handleChange = (event) => {
-    this.syncHeightWithShadow(event.target.value);
+    if ( !this.props.hasOwnProperty('value') ) {
```

Comment:
But, if the value is set as a falsy value, for example "" (an empty string), then `if (!this.props.value) {` wouldn't work.